src/application/services/useNoteSettings.ts

```diff
@@ -71,7 +71,11 @@ export default function (): UseNoteSettingsComposableState {
    * @param data - Note settings data with new values
    */
   const update = async (id: NoteId, data: Partial<NoteSettings>): Promise<void> => {
-    noteSettings.value = await noteSettingsService.patchNoteSettingsByNoteId(id, data);
+    const response = await noteSettingsService.patchNoteSettingsByNoteId(id, data);
+
+    for (let key in noteSettings.value) {
+      key = response[key as keyof typeof noteSettings.value] as string;
+    }
```

Comment:
seems unsafe. Iterate over response keys and check if note settings have the same key